packages/shared/tsconfig.json

```diff
@@ -98,7 +98,9 @@
   },
   "include": [
     "./src/**/*"
-  ],
+,
+"../server/src/transaction-context"
+],
```

Comment:
Sorry I'm late to this PR, but I'm a bit confused by this.

It seems odd (and maybe even problematic?) that we need to compile source from server in shared.